### ISSUE TITLE
Tooltip present on "save as json", "save as excel", "save as CSV" buttons are not accessible with keyboard: A11y_mssql for VSCode_mssql Extension_Select top 1000 Results window_Keyboard 

### ISSUE THREAD

**Issue by aniketr99** [microsoft/vscode-mssql#17875]
## GitHubTags: 
#A11yTCS;#Win11;#DesktopApp;#BM_mssql for VSCode_Win32_Feb2024;#A11ySev2;#mssql for VSCode;#WCAG2.1.1;#A11yMAS;#Win32;#AINotInScope;#Keyboard;

## Environment Details:
Application Name: mssql for VSCode
Operating System: Win11[23H2]
OS Build: 26016.1012
Visual studio code: 1.86.2
Extension Version: v1.22.1

## Repro Steps:
1. Open Visual studio code
2. Tab till extension button and search for "mssql".
3. Tab till install and move to the SQL Server button.
4. Tab till Add Connection Button and insert the valid credentials.
5. Tab till the database and hit enter.
6. Tab till the System database and hit enter.
7. Tab till the Tables and hit enter.
8. Tab till any table under System tables and hit shift + F10
9. Tab till Select Top 1000 and hit enter
10. Tab till Execute Query and hit enter
11. Tab till "Results" dropdown and hit enter
12. Tab till  "save as json", "save as excel", "save as CSV" buttons and verify whether tooltip is accessible with keyboard or not.

## Actual:
Tooltip present on "save as json", "save as excel", "save as CSV" buttons are not accessible with keyboard.

## Expected:
Tooltip present on "save as json", "save as excel", "save as CSV" buttons should be accessible with keyboard.

## User Impact:
Keyboard users will not get about the controls if Tooltips are not appearing when controls are receiving focus.

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Attachment
![Bug_17875_Keyboard](https://github.com/microsoft/vscode-mssql/assets/100269580/2977c722-4e71-42a5-b5a5-22b2a56b4bce)

**Comment by alanrenmsft** [comment]
@aniketr99 please refer to https://github.com/microsoft/vscode-mssql/issues/17201 and downgrade this one to sev3. thanks.

**Comment by aniketr99** [comment]
@alanrenmsft, please join office hours to discuss this bug. Thanks!

**Comment by alanrenmsft** [comment]
@laurennat

**Comment by aniketr99** [comment]
Tags:
#Regressed:08-12-24;
Regressed the bug in the latest environment:
Issue is not fixed. Attachment is added for the reference.
![Presentation1](https://github.com/user-attachments/assets/9dbd1a19-a614-4291-b9d7-05fbe2b3663c)

**Comment by kburtram** [comment]
@laurennat it looks like the tester said this issue isn't fixed.  Could you please follow up to get this issue resolved?

**Comment by laurenastrid1** [comment]
This has been fixed for a while, I just need a tester to verify: ![Image](https://github.com/user-attachments/assets/a7392c32-457e-4a0e-8c9b-80b02b8a3eec)

**Comment by kburtram** [comment]
@laurenastrid1 could you please follow up with the tester to get this bug and the other accessibility issue closed?  And confirm that the compliance scorecard reflects these fixes?  These bugs are now out of SLA and impacting our accessibility grade at this point.  Thanks!

**Comment by ManishaC1** [comment]
@kburtram @laurenastrid1 

Verified the issue in below version of Azure Data Studio insiders. Issue is still reproducing in the environment. Please refer the attached

![Image](https://github.com/user-attachments/assets/71825952-e099-47a4-aadf-e6b8fc8fe1af)

**Comment by laurenastrid1** [comment]
Yes, will do.

**Comment by laurennat** [comment]
This is now fixed, please email/Teams me to get the new vsix with the fix @ManishaC1

**Comment by ManishaC1** [comment]
Issue is fixed.

VS Code version: 1.97.0-insider (user setup)
Commit: a74aabd9bb87c33b047c822aa79d265bc5f5543e
Date: 2025-01-17T05:04:33.457Z
Electron: 32.2.7
ElectronBuildId: 10660205
Chromium: 128.0.6613.186
Node.js: 20.18.1
V8: 12.8.374.38-electron.0
OS: Windows_NT x64 10.0.26100

Extension used: mssql-1.28.0 1.vsix

![Image](https://github.com/user-attachments/assets/00cf055e-a4bd-47de-9ccd-dec45eb24f46)

**Comment by laurennat** [comment]
@ManishaC1 asked me to close it
